src/test/java/eu/europeana/iiif/ExampleData.java

```diff
@@ -20,4 +20,6 @@
     public static final String EXAMPLE_FULLTEXT_ID = "/9200396/BibliographicResource_3000118436342";
     public static final String EXAMPLE_FULLTEXT_PAGENR = "1";
 
+    public static String EXAMPLE_FULLTEXT_SUMMARY_RESPONSE = "{\"items\":[{\"id\":\"https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436342/canvas/1\",\"type\":\"Canvas\",\"annotations\":[{\"id\":\"https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436342/canvas/1?language=fi\",\"type\":\"AnnotationPage\",\"language\":\"fi\"},{\"id\":\"https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436342/canvas/1?language=se\",\"type\":\"AnnotationPage\",\"language\":\"se\"}]}]}";
+    public static final String EXAMPLE_FULLTEXT_SUMMARY_FRAGMENT = "https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436342/canvas/1?language=fi";
```

Comment:
In the annotationPage types, the `/canvas` part in their id was changed to `/annopage/` and the parameter was changed to `'lang` instead of `language` because the id needs to point to the actual translation annopage.
I updated this locally in this definition, but then the ManifestServiceTest.testFullSummaryExists() fails, so I don't think this will work in practice.

Reply:
Ah, I see, the problem was the regex in FullTextSummaryCanvas:
private static final Pattern PAGENUMBERPATTERN = Pattern.compile("/canvas/(\\d+)$");
which I now changed to
private static final Pattern PAGENUMBERPATTERN = Pattern.compile("/annopage/(\\d+)$");